### PR TITLE
add alt to img

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/issues/347

### CODE REVIEW

File: themes/ocmTheme/layouts/partials/supports.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <section class="supports-section">
     <h1 class="text-center feature-h1">{{ i18n "supported_by" }}</h1>
     <div class="supports-grid">
-        <a class="supports-logo" href="https://www.alibabacloud.com/" title="Alibab Cloud" target="_blank"><img src="/alibaba-cloud.svg" /></a>
-        <a class="supports-logo" href="https://www.antgroup.com/" title="Ant Group" target="_blank"><img src="/ant-group.svg" /></a>
-        <a class="supports-logo" href="https://www.redhat.com/" title="Red Hat" target="_blank"><img src="/red-hat.svg" /></a>
-        <a class="supports-logo tencent-logo" href="https://cloud.tencent.com/" title="Tencent Cloud" target="_blank"><img src="/tencent-cloud.svg" /></a>
-        <a class="supports-logo" href="https://www.xiaohongshu.com/" title="Xiaohongshu" target="_blank"><img src="/xiaohongshu.svg" /></a>
+        <a class="supports-logo" href="https://www.alibabacloud.com/" title="Alibab Cloud" target="_blank"><img src="/alibaba-cloud.svg" alt="Alibaba cloud" /></a>
```
Comment (mikeshng) [inline themes/ocmTheme/layouts/partials/supports.html:5]:
```suggestion
        <a class="supports-logo" href="https://www.alibabacloud.com/" title="Alibab Cloud" target="_blank"><img src="/alibaba-cloud.svg" alt="Alibaba Cloud" /></a>
```

File: themes/ocmTheme/layouts/partials/supports.html [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
+        <a class="supports-logo" href="https://www.antgroup.com/" title="Ant Group" target="_blank"><img src="/ant-group.svg" alt="Ant group" /></a>
```
Comment (mikeshng) [inline themes/ocmTheme/layouts/partials/supports.html:6]:
```suggestion
        <a class="supports-logo" href="https://www.antgroup.com/" title="Ant Group" target="_blank"><img src="/ant-group.svg" alt="Ant Group" /></a>
```

File: themes/ocmTheme/layouts/partials/supports.html [hunk 3]
```diff
@@ -2,10 +2,10 @@
...
+        <a class="supports-logo" href="https://www.redhat.com/" title="Red Hat" target="_blank"><img src="/red-hat.svg" alt="Red hat" /></a>
```
Comment (mikeshng) [inline themes/ocmTheme/layouts/partials/supports.html:7]:
```suggestion
        <a class="supports-logo" href="https://www.redhat.com/" title="Red Hat" target="_blank"><img src="/red-hat.svg" alt="Red Hat" /></a>
```

File: themes/ocmTheme/layouts/partials/supports.html [hunk 4]
```diff
@@ -2,10 +2,10 @@
...
+        <a class="supports-logo tencent-logo" href="https://cloud.tencent.com/" title="Tencent Cloud" target="_blank"><img src="/tencent-cloud.svg" alt="Tencent cloud" /></a>
```
Comment (mikeshng) [inline themes/ocmTheme/layouts/partials/supports.html:8]:
```suggestion
        <a class="supports-logo tencent-logo" href="https://cloud.tencent.com/" title="Tencent Cloud" target="_blank"><img src="/tencent-cloud.svg" alt="Tencent Cloud" /></a>
```

**Review by mikeshng — CHANGES_REQUESTED** [review]
Thank you for your contribution! 
Just some minor nits.

File: themes/ocmTheme/layouts/partials/supports.html (hunk 1 above)
Comment (Arhell) [reply themes/ocmTheme/layouts/partials/supports.html]: Done

File: themes/ocmTheme/layouts/partials/supports.html (hunk 2 above)
Comment (Arhell) [reply themes/ocmTheme/layouts/partials/supports.html]: Done

File: themes/ocmTheme/layouts/partials/supports.html (hunk 3 above)
Comment (Arhell) [reply themes/ocmTheme/layouts/partials/supports.html]: Done

File: themes/ocmTheme/layouts/partials/supports.html (hunk 4 above)
Comment (Arhell) [reply themes/ocmTheme/layouts/partials/supports.html]: Done

**Comment by mikeshng** [comment]
/approve

@chenz4027 please take a quick look whenever you have some cycles. Thanks!

**Pull request merged**